index.bs

```diff
@@ -122,16 +122,20 @@ and |options| is invoked, the user agent MUST run the following steps:
 
 1. If the algorithm is not <a>triggered by user activation</a>, throw a
     {{SecurityError}} and abort these steps.
-2. Let |pipWindow| be the Picture In Picture window that is <dfn>always on
+2. If document is not allowed to use the <a>policy-controlled feature</a> named
+    <code>"picture-in-picture"</code>, throw a {{SecurityError}} and abort these
+    steps.
+3. Let |pipWindow| be the Picture In Picture window that is <dfn>always on
     top</dfn>. It consistently stays above most other windows.
-3. Let |pipVideo| be the video contained in the |pipWindow|.
-4. If a |pipWindow| is not already created, create one.
-5. If one can't be created, throw a {{UnknownError}} and abort these steps.
-6. Apply |options| {{width}}, {{height}}, {{PictureInPictureOptions/top}}, and
+4. Let |pipVideo| be the video contained in the |pipWindow|.
+5. If a |pipWindow| is not already created, create one.
+6. If one can't be created because user agent blocked feature for instance,
+    throw a {{NotAllowedError}} and abort these steps.
+7. Apply |options| {{width}}, {{height}}, {{PictureInPictureOptions/top}}, and
     {{start}} to the |pipWindow| dimensions and position. If some |options|
     can't be applied, it SHOULD apply latest ones that were successfully set if
```

Comment:
If we add "Optionnaly", maybe the SHOULD could be dropped?

Reply:
Done.